rotkehlchen/db/dbhandler.py

```diff
@@ -2046,7 +2046,14 @@ def get_exchange_credentials(
                 continue
 
             passphrase = None if entry[4] is None else entry[4]
-            location = Location.deserialize_from_db(entry[1])
+            try:
+                location = Location.deserialize_from_db(entry[1])
+            except DeserializationError as e:
+                self.msg_aggregator.add_error(
+                    f'Found unknow location {entry[1]} for exchange {entry[0]}. '
+                    f'This means that you used a newer version of the app. {str(e)}',
```

Comment:
you can't be sure about what this means.


```suggestion
                    f'Found unknown location {entry[1]} for exchange {entry[0]} at get_exchange_credentials. '
                    f'This could mean that you are opening the app with an older version. {str(e)}',
```